api/SnapEvent.json

```diff
@@ -35,42 +35,6 @@
           "deprecated": false
         }
       },
-      "SnapEvent": {
-        "__compat": {
-          "description": "`SnapEvent()` constructor",
-          "mdn_url": "https://developer.mozilla.org/docs/Web/API/SnapEvent/SnapEvent",
-          "spec_url": "https://drafts.csswg.org/css-scroll-snap-2/#dom-snapevent-snapevent",
-          "tags": [
-            "web-features:scroll-snap-events"
-          ],
-          "support": {
-            "chrome": {
-              "version_added": "129"
-            },
```

Comment:
It is now behind the `#enable-experimental-web-platform-features` flag since Chrome 138 (see https://github.com/chromium/chromium/commit/d7db7ce142981af9d65339bab3ce25c6b21e71bb), so let's update the data instead of removing!